src/Frontend/Context/category.js

```diff
@@ -22,7 +23,7 @@ const Category = createContext({ categories: [] });
 
 export const CategoryProvider = ({ children }) => {
   const CategoryReducer = (
```

Comment:
You can move this Reducer to a Separate File, for the better convention.

Reply:
Will do in refactoring and still category context code is there to write for me
I have kept a threshould if the file increases 300 lines of code then we will wap in seperate modules/files